src/mlpack/core/data/normalize_labels_impl.hpp

```diff
@@ -39,32 +40,32 @@ void NormalizeLabels(const RowType& labelsIn,
   // we'll resize it back down to its actual size.
   mapping.set_size(labelsIn.n_elem);
   labels.set_size(labelsIn.n_elem);
+  // Map for mapping labelIn to their label
+  std::map<eT, size_t> hastTable;
```

Comment:
I'd suggest calling this `labelMap`---it makes it a little more clear what its function is.

Reply:
renamed it as labelmap